frameworks/digital-outcomes-and-specialists-2/search_mappings/briefs.json

```diff
@@ -62,6 +62,28 @@
               "target_field": "idHash"
             }
           },
+          {
+            "set_conditionally": {
+              "field": "status",
+              "target_field": "statusOpenClosed",
+              "any_of": [
+                "live"
+              ],
+              "set_value": "open"
+            }
+          },
+          {
+            "set_conditionally": {
+              "field": "status",
+              "target_field": "statusOpenClosed",
+              "any_of": [
+                "awarded",
+                "unsuccessful",
+                "cancelled"
+              ],
+              "set_value": "closed"
+            }
+          },
           {
             "set_conditionally": {
```

Comment:
We're still mapping the statuses to closed, below? Should we remove that?

Reply:
(Ah - I see this is stage 1!)

Reply:
These will be removed a little bit later. Doubling up momentarily will allow us not to break buyer-frontend's search filters